### PR TITLE
fix #76631 dlford.io

### LINKED ISSUE #76631: dlford.io
### Issue URL (Annoyance)
 [https://www.dlford.io/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.dlford.io%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#76631

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 balcaorural.com.br##.j-cookieAlert
+dlford.io###gatsby-focus-wrapper > div > footer
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:95]: It's better to use ext-css here

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:95]: No need. I tried no FP.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:95]: @Sergey-Lyapin Fixed.

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: In both cases footer is broken
image
JS rule can be used here to work with LocalStorage(you can find similar rules in Annoyances filter)
image

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 balcaorural.com.br##.j-cookieAlert
+dlford.io###gatsby-focus-wrapper > div > footer:contains(Privacy Notice)
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:95]: please change to:
dlford.io#?##gatsby-focus-wrapper > div[class^="layout__Outer-sc-"] > footer:has(> section > p > div:contains(Privacy Notice))

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt]: `dlford.io#%#(function() { try { if ('undefined' != typeof localStorage) { localStorage.setItem('closedPrivacyNotice', 'true') } } catch (ex) {} })();`

**Comment by wout-codes** [comment]
Okay. Fixed.

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @Sergey-Lyapin Where did you find this? `() { try { if ('undefined' != typeof localStorage) {`? Isn't this enough? `localStorage.setItem('closedPrivacyNotice', 'true')`? That's a pure javascript.

**Pull request merged**